docs/DRONECAN.md

```diff
@@ -10,19 +10,37 @@ TBD
 
 ## mLRS Receiver Settings
 
+#### RC
+
 With a CAN-enabled firmware flashed, the receiver will output RC channel data on the CAN bus without any further settings.
 
-In order to enable also the serial (MAVLink) communication via DroneCAN, one needs to set "Rx Serial Port = can".
+#### MAVLink
+
+In order to enable also the serial (MAVLink) communication via DroneCAN, one needs to set:
```

Comment:
Nitpick, prefer 'In order to enable serial (MAVLink) communication via DroneCAN, one needs to set:'